README.md

```diff
@@ -72,6 +73,7 @@ The `options` object has the following properties:
 - `monthNames`: month names
 - `minDate`: minimum selectable date (there is a setter on Calendar object, too)
 - `maxDate`: maximum selectable date (there is a setter on Calendar object, too)
+- `shortcutButtons`: list of additional buttons data, which will be displayed at the top of calendar (example: [{"label": "Today", "differenceCurrentDate": 0}])
```

Comment:
Please, use backticks ` to display code in a README.
Moreover add an example using the new API:

```suggestion
- `shortcutButtons`: list of additional buttons data, which will be displayed at the top of calendar. You can add a button with: `shortcutButtons: [{"label": "Today", "action": "ping"}]` and you can handle it with `bot.action("ping", context => context.reply("pong"))`
```